xrpl/models/transactions/credential_create.py

```diff
@@ -0,0 +1,76 @@
+"""Model for CredentialCreate transaction type."""
+
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+from typing_extensions import Self
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import (
+    HEX_REGEX,
+    KW_ONLY_DATACLASS,
+    _get_credential_type_error,
+    require_kwargs_on_init,
+)
+
+_MAX_URI_LENGTH = 256
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class CredentialCreate(Transaction):
+    """This transaction creates a Credential object. It must be sent by the issuer."""
+
+    transaction_type: TransactionType = field(
+        default=TransactionType.CREDENTIAL_CREATE,
+        init=False,
+    )
+
+    account: str = REQUIRED  # type: ignore
+    """
+    The issuer of the credential.
+    """
+    subject: str = REQUIRED  # type: ignore
```

Comment:
Please add empty lines between the fields

Reply:
c27ab41b9dd6194250f49c10abe82438f0882903